rust/kernel/mm.rs

```diff
@@ -147,3 +148,32 @@ pub mod virt {
         pub const MERGEABLE: usize = bindings::VM_MERGEABLE as _;
     }
 }
+
+/// Wraps the kernel's `struct mm_struct`.
+#[repr(transparent)]
+pub struct MmStruct(pub(crate) UnsafeCell<bindings::mm_struct>);
```

Comment:
`MmStruct` is not a good name. It stutters the module name: `mm::MmStruct` and is followed by a generic "Struct".

Perhaps just `Mm` would be ok because we have precedents everywhere for repeating the module name, plus it reflects the C name (albeit with different casing).